Sources/UIScrollView.swift

```diff
@@ -120,10 +127,24 @@ open class UIScrollView: UIView {
     open var contentInset: UIEdgeInsets = .zero
     open var contentSize: CGSize = .zero
 
-
     // MARK: Scroll Indicators
 
     let indicatorThickness: CGFloat = 2.5
+    
+    public var indicatorStyle: UIScrollViewIndicatorStyle = .`default` {
```

Comment:
`UIScrollViewIndicatorStyle` is now `UIScrollView.IndicatorStyle`: https://developer.apple.com/documentation/uikit/uiscrollview/indicatorstyle

Reply:
done